soc/arm/st_stm32/common/Kconfig.defconfig.series

```diff
@@ -113,6 +113,9 @@ if DMA
 config DMA_STM32
 	default y
 
+config HEAP_MEM_POOL_SIZE
```

Comment:
Putting this directly in `drivers/dma/Kconfig.stm32 ` would help finding the info faster.

Reply:
Since symbol has to be in a defconfig this file is a good choice.